mmdet3d/datasets/pipelines/loading.py

```diff
@@ -19,6 +23,9 @@ class LoadMultiViewImageFromFiles(object):
     """
 
     def __init__(self, to_float32=False, color_type='unchanged'):
+        warnings.warn('DeprecationWarning: LoadMultiViewImageFromFiles'
+                      'is deprecated, please use MultiViewPipeline('
+                      'transforms=[dict(type="LoadImageFromFile")]) instead')
```

Comment:
Whether to deprecate this function will be further discussed. Maybe we can temporarily remove this warning and TODO at line 12.

Reply:
Ok, will remove it.